ORK1Kit/ORK1Kit/ActiveTasks/CEV1RangeOfMotionStepViewController.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  CEV1RangeOfMotionStepViewController.swift
+//  ORK1Kit
+//
+//  Created by Eric Schramm on 3/27/23.
+//  Copyright © 2023 researchkit.org. All rights reserved.
+//
+
+import Foundation
+
+public class CEV1RangeOfMotionStepViewController: ORK1RangeOfMotionStepViewController {
+    
+    let debugMode = false
+    
+    var referenceAttitude: CMAttitude?
+    var highestAngle = Double.leastNonzeroMagnitude
+    var lowestAngle = Double.greatestFiniteMagnitude
+    var lastRawAngle: Double?
+    var motionUpdates = 0
+    var clockwiseCrossesZero = 0
+    var cumulativeAngle = 0.0
+    var firstRawAngle = 0.0
+    
+    public override var result: ORK1StepResult? {
+        let stepResult = super.result
+        
+        guard let step = step as? CEV1RangeOfMotionStep else {
+            return nil
+        }
+    
+        let result = ORK1RangeOfMotionResult(identifier: step.identifier)
+        switch step.motionType {
+        case .flexion:
+            switch step.measuredDirectionOfRotation {
+            case .clockwise:
+                result.extended = lowestAngle
+                result.flexed = highestAngle
+            case .counterClockwise:
```

Comment:
Actually, I never finished some cleanup here from a refactor. The switch is no longer needed because we measure the angle by adding the changes to the starting angle. Because we also know the expected direction of rotation, we know whether to add or subtract the delta. Because of that, the lowest angle will always be the starting angle and the highest the ending angle when measuring along the direction of rotation. So we only need to differ based on whether the movement is considered a flexion or extension which defines the starting/ending (lowest/highest) angle.

Reply:
Latest commit with switch statement removed looks good